Update model possibilities (#51)
runwayml/learn: docs/create-with-runwayml/generate.md

# Generate

> The pre-trained models in this category generate synthetic output based the training data, such as photorealistic images and coherent text.

Reviewer: ```suggestion
The pre-trained models in this category can generate synthetic outputs based on specific training data, such as photorealistic images and coherent text.
```

> Note: models are added to the RunwayML app often, and this page does not necessarily respresent the full list. Contributions, demos, and tutorials are welcome here!

Reviewer: ```suggestion
> **Note**: models are added to the RunwayML app often, and this page does not necessarily represent the full list. Contributions, demos, and tutorials are welcome here!
```

## Generate Images from Text

> | Model | Description |  |

Reviewer: ```suggestion
| Model | Description | More |
```

| :--- | :---| :--- |
| [AttnGAN](https://open-app.runwayml.com/?model=runway/AttnGAN) | Turn text descriptions of scenes into synthesized images | [Tutorial](tutorials/tutorial_t2i.md) |

## Generate Text from Images

> | Model | Description |  |

Reviewer: ```suggestion
| Model | Description | More |
```

| :--- | :---| :--- |
| [im2txt](https://open-app.runwayml.com/?model=runway/im2txt) | Generate sentence descriptions of images | [Tutorial](tutorials/tutorial_im2txt.md) |

## Generate Text

> | Model | Description |  |

Reviewer: ```suggestion
| Model | Description | More |
```

| :--- | :---| :--- |
| [GPT-2](https://open-app.runwayml.com/?model=runway/GPT-2) | Generate paragraphs of text from a prompt | |
| [OpenGPT-2](https://open-app.runwayml.com/?model=runway/OpenGPT-2) | Replication effort of GPT-2's largest model (1.5B parameters) | |
| [universal-sentence-encoder-xling-runway](https://open-app.runwayml.com/?model=aparrish/universal-sentence-encoder-xling-runway) | Encodes sentences to fixed-length vectors | |
| [vae-lagging-encoder-poetry](https://open-app.runwayml.com/?model=aparrish/vae-lagging-encoder-poetry) | Gutenberg Poetry Variational Autoencoder | |

## Synthesize Images

> | Model | Description |  |

Reviewer: ```suggestion
| Model | Description | More |
```
